include/Util/TransparentImage.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef UTIL_TRANSPARENT_IMAGE_HPP
+#define UTIL_TRANSPARENT_IMAGE_HPP
+
+#include <string>
+#include <SDL.h>
+#include <SDL_image.h>
+#include <iostream>
+#include <Util/Logger.hpp>
+
+#include "Core/TransparentImageBase64.hpp"
+#include "Base64.hpp"
+
+static constexpr auto TRANSPARENT_IMAGE_BASE64_DECODE_LENGTH = DecodeBase64Length(TRANSPARENT_IMAGE);
+static constexpr auto TRANSPARENT_IMAGE_BASE64_DECODE = DecodeBase64<TRANSPARENT_IMAGE_BASE64_DECODE_LENGTH>(TRANSPARENT_IMAGE);
+
+SDL_Surface* GetTransparentImageSDLSurface(){
+    SDL_RWops* rwop = SDL_RWFromConstMem(TRANSPARENT_IMAGE_BASE64_DECODE.data(), TRANSPARENT_IMAGE_BASE64_DECODE.size());
+    SDL_Surface* aSurface = IMG_LoadTyped_RW(rwop, 1, "PNG");
+
+    if (aSurface == nullptr) {
+        LOG_ERROR("base64ToSurface");
+    }
+
+    return aSurface;
+}
+
+#endif
```

Comment:
Blank line at EOF as mentioned in [Code Styles and Guidelines](https://hackmd.io/bOzigaM9R5Sgsl0nqMtRQg).

Reply:
Fix on 9958cda